content/community/user_groups.adoc

```diff
@@ -0,0 +1,149 @@
+= Clojure user groups
+2018-04-06
+:type: community
+:toc: macro
+:icons: font
+
+ifdef::env-github,env-browser[:outfilessuffix: .adoc]
+
+Do you wish there was a Clojure User Group in your area? Check
+out our page of <<xref/../start_group,tips for starting your own>>
+
+If you'd like to have your group listed here, please submit a PR against the
+https://gitub.com/clojure/clojure-site[clojure-site] github repo.
+
+Join the https://groups.google.com/forum/?fromgroups#!forum/clojure-power[clojure-power]
+mailing list for group organizers!
+
+[ <<xref/../user_groups#americas,Americas>>] [<<xref/../user_groups#europe,Europe>>]
```

Comment:
links can just be <<user_groups#americas,Americas>> etc I think, some extra spaces in here too